src/model/remoteConfig.ts

```diff
@@ -52,6 +55,7 @@ interface RainbowConfig extends Record<string, string | boolean | number> {
   op_chains_enabled: boolean;
   mainnet_enabled: boolean;
   goerli_enabled: boolean;
+  avax_enabled: boolean;
```

Comment:
smol nit but feel like this should be `avalanche_enabled`
```suggestion
  avalanche_enabled: boolean;
```